forgot_password/handlers/verify_code.py

```diff
@@ -120,6 +123,27 @@ def before_plugin_ready(*args, **kwargs):
         if settings.modify_acl:
             schema_add_key_verified_acl(managed_flags)
 
+    @skygear.op('user:verify_request:test', key_required=True)
+    def test_verify_request_lambda(record_key,
+                                   record_value,
+                                   provider_settings={}):
```

Comment:
Um... this is interesting, especially when the lambda is not protected with master key. Correct me if I am wrong, but  I suppose this allows the caller of the lambda to change the provider settings, which include SMTP or SMS settings.

I guess you want to change the provider settings here because you don’t want to restart the server just to test changes.

If restarting the server to test sending email/SMS is not accepted, I think this lambda should be protected with master key.